role_assignments.go

```diff
@@ -0,0 +1,47 @@
+package gapi
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type RoleAssignments struct {
+	RoleUID         string           `json:"role_uid"`
+	Users           []RoleAssignment `json:"users,omitempty"`
+	Teams           []RoleAssignment `json:"teams,omitempty"`
+	ServiceAccounts []RoleAssignment `json:"service_accounts,omitempty"`
+}
+
+type RoleAssignment struct {
+	ID     int  `json:"id"`
+	Global bool `json:"global"`
+}
+
+func (c *Client) GetRoleAssignments(uid string) (*RoleAssignments, error) {
+	assignments := &RoleAssignments{}
+	url := fmt.Sprintf("/api/access-control/roles/%s/assignments", uid)
+	if err := c.request(http.MethodGet, url, nil, nil, assignments); err != nil {
+		return nil, err
+	}
+
+	return assignments, nil
+}
+
+func (c *Client) UpdateRoleAssignments(ra RoleAssignments) (*RoleAssignments, error) {
```

Comment:
```suggestion
func (c *Client) UpdateRoleAssignments(ra *RoleAssignments) (*RoleAssignments, error) {
```

nit: might be a bit more efficient here